testsuite/MDAnalysisTests/coordinates/test_crd.py

```diff
@@ -89,23 +82,23 @@ class TestCRDWriterMissingAttrs(object):
                  'tempfactors': 0.0,
                  }
 
-    def _check_warns(self, missing_attr):
+    @pytest.mark.parametrize('missing_attr', req_attrs)
```

Comment:
Yeah sure, that'd work too.  Just something with a deterministic order, else "different" tests get collected by two processes.